### PR TITLE
feat(general): add banner to announce new learning platform

### PR DESCRIPTION
**Pull request by y4izus**
## Changes

Fix https://github.com/Qiskit/qiskit.org/issues/3410

## Implementation details

Adds and styles the banner to announce the new learning platform

## How to read this PR

Check that the banner is working across the pages under this repo. It is not implemented on the Metal and Error pages because they are not related with Learning

## Screenshots

Captura de pantalla 2023-07-07 a las 17 26 42

### CODE REVIEW

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <qiskit-banner>
+    Introducing&nbsp;
+    <a href="#">IBM Quantum Learning</a>, a new home for free quantum education
+    resources. Continue to use Learn on Qiskit until December 1st, 2023
```
Comment (techtolentino) [inline components/Layout/Banner/LayoutBannerNewLearningPlatform.vue:5]: should this end with a `.`

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+qiskit-banner {
+  display: inline-block;
+  width: 100%;
+  background: carbon.$cool-gray-20 0%;
+  color: inherit;
+
+  a,
```
Comment (techtolentino) [inline components/Layout/Banner/LayoutBannerNewLearningPlatform.vue:18]: normally, if I recall, we don't style main html elements `<p>` or `<div>` or `<a>` tags, for example. Instead, we _usually_ give them class names.

is there a reason we're not using that approach here?

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  </qiskit-banner>
+</template>
+
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+qiskit-banner {
+  display: inline-block;
+  width: 100%;
+  background: carbon.$cool-gray-20 0%;
```
Comment (techtolentino) [inline components/Layout/Banner/LayoutBannerNewLearningPlatform.vue:15]: any reason to not use `background-color: carbon.$cool-gray-20;`?

**Review by techtolentino — COMMENTED** [review]
I wasn't sure if this was ready for review, but I saw the PR status and decided to give it a look.

Just some minor questions.

Otherwise, it looks good!

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue (hunk 2 above)
Comment (y4izus) [reply components/Layout/Banner/LayoutBannerNewLearningPlatform.vue:18]: yes, it's overwritting the web-component styles and that's what is used there

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue (hunk 1 above)
Comment (y4izus) [reply components/Layout/Banner/LayoutBannerNewLearningPlatform.vue]: yes, thank you! Done in https://github.com/Qiskit/qiskit.org/pull/3411/commits/f881264b2497b8eed4d35f56368eadcdb8e07060

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue (hunk 3 above)
Comment (y4izus) [reply components/Layout/Banner/LayoutBannerNewLearningPlatform.vue]: good catch! Updated here https://github.com/Qiskit/qiskit.org/pull/3411/commits/f881264b2497b8eed4d35f56368eadcdb8e07060

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <qiskit-banner>
+    Introducing&nbsp;
+    <a href="#">IBM Quantum Learning</a>, a new home for free quantum education
+    resources. Continue to use Learn on Qiskit until December 1st, 2023.
```
Comment (1ucian0) [inline components/Layout/Banner/LayoutBannerNewLearningPlatform.vue:5]: Small change in the wording, so it is clear what  "Learn on Qiskit" means.
```suggestion
    resources. Continue to use qiskit.org/learn until December 1st, 2023.
```

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue (hunk 2 above)
Comment (techtolentino) [reply components/Layout/Banner/LayoutBannerNewLearningPlatform.vue:18]: make sense - thanks for pointing this out

File: components/Layout/Banner/LayoutBannerNewLearningPlatform.vue (hunk 4 above)
Comment (y4izus) [reply components/Layout/Banner/LayoutBannerNewLearningPlatform.vue]: Done https://github.com/Qiskit/qiskit.org/pull/3411/commits/4015815a22642e2c0c6a2c519fd0f2d57bb3c4b4

Comment (1ucian0) [reply components/Layout/Banner/LayoutBannerNewLearningPlatform.vue]: cc @JRussellHuffman @pandasa123

**Comment by 1ucian0** [comment]
Additionally, those moved courses can be annotated like this, where the second link points to https://github.com/Qiskit/textbook/tree/main/notebooks/<something>

Screenshot 2023-07-12 at 15 37 28

**Comment by 1ucian0** [comment]
Is this changing affecting learn.qiskit.org too? It is particularly important in https://learn.qiskit.org/signin, as the sunset will/might affect the course progress of the users.

**Review by 1ucian0 — APPROVED** [review]

**Pull request merged**